src/Request/RectorRunFormRequest.php

```diff
@@ -25,38 +25,38 @@ public function authorize(): bool
     public function rules(): array
     {
         $shortPhpContentsRule = $this->make(ShortPhpContentsRule::class);
-        $validPhpSyntaxRule = $this->make(ValidPhpSyntaxRule::class);
+        $validAndSafePhpSyntaxRule = $this->make(ValidAndSafePhpSyntaxRule::class);
+
+        // @todo list forbidden functions? merge into @see ValidAndSafePhpSyntaxRule
```

Comment:
I am doing this on wip PR:

- https://github.com/rectorphp/getrector-com/pull/2414

Reply:
Awesome :+1: I'll leave that to you then